frontend/app/components/work-packages/wp-display-attr/wp-display-attr.directive.ts

```diff
@@ -110,20 +112,18 @@ export class WorkPackageDisplayAttributeController {
   }
 }
 
-function wpDisplayAttrDirective() {
+function wpDisplayAttrDirective(wpCacheService:WorkPackageCacheService) {
 
   function wpTdLink(scope,
                     element,
                     attr,
                     controllers) {
 
-    // Listen for changes to the work package on the form ctrl
-    var formCtrl = controllers[0];
```

Comment:
Good one, I missed that we don't require it anywhere else in the display directive.